src/layout/Tabbed.js

```diff
@@ -20,6 +20,7 @@
     Tabbed.prototype.publish("activeTabIdx", 0, "number", "Index of active tab", null, {});
 
     Tabbed.prototype.publish("labels", [], "array", "Array of tab labels sharing an index with ", null, { tags: ["Private"] });
+    Tabbed.prototype.publish("tabsOnBottom", false, "boolean", "Position the tabs at the bottom of the widget", null, { tags: ["Private"] });
```

Comment:
make this a set called "tabLocation",   ["top", "bottom"] with "top" the default (in case we need left/right in the future